src/main/java/edu/texas/social_computing/hospitals/FileImporter.java

```diff
@@ -0,0 +1,70 @@
+package edu.texas.social_computing.hospitals;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+class FileImporter {
+    private static Scanner getScanner(String path) throws FileNotFoundException {
+        File file = new File(path);
+        Scanner sc;
+        try {
+            sc = new Scanner(file);
+        } catch (FileNotFoundException e) {
+            System.out.println("couldn't find file" + e);
+            throw e;
```

Comment:
So here (which I'm gonna fix in a quick push), there isn't really a point in the entire try/catch since you're just rethrowing the exception anyways. So either this will crash the program, and the exception will get printed out, or some other method will have to catch and deal with it, so they should do the logging.